plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/MetastoreValidator.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg.catalog;
+
+import com.google.inject.Inject;
+import io.trino.plugin.hive.metastore.cache.SharedHiveMetastoreCache;
+
+public class MetastoreValidator
```

Comment:
The "Extract MetastoreValidator" commit adds a new class, but doesn't delete any existing code.
What am i missing?

Reply:
I am actually missing attention.
Thank you for pointing out this problem.